ts/screens/wallet/payment/ConfirmPaymentMethodScreen.tsx

```diff
@@ -249,7 +249,7 @@ const mapDispatchToProps = (dispatch: Dispatch, props: OwnProps) => {
         })
       );
       // reset the payment state
-      dispatch(paymentInitializeState());
+      // dispatch(paymentInitializeState());
```

Comment:
if you comment a line you should live a comment why it is commented
otherwise the line could be deleted